build.sbt

```diff
@@ -210,6 +210,7 @@ lazy val nodeClient =  project
 
     npmIClient := {
       Process(s"npm install @aml-org/amf-custom-validator@$amfCustomValidatorJSVersion", new File("./als-node-client/node-package/")) #&&
+      Process(s"cp -r ../../als-server/js/node-package/typescript/als-server.d.ts ./typescript/als-server.d.ts", new File("./als-node-client/node-package/")) #&&
```

Comment:
Copies and keeps typings file for `als-node-client` updated

Reply:
Maybe we should update it manually, but I can guarantee we would forget to do so.